homeassistant/components/vizio/config_flow.py

```diff
@@ -394,43 +423,26 @@ def __init__(self) -> None:
             errors=errors,
         )
 
-    async def async_step_pairing_complete_import(
-        self, user_input: Dict[str, Any] = None
-    ) -> Dict[str, Any]:
-        """Complete import config flow by displaying final message to show user access token and give further instructions."""
+    async def _pairing_complete(self, step_id: str) -> Dict[str, Any]:
+        """Handle config flow completion."""
         if not self._must_show_form:
             return await self._create_entry_if_unique(self._data)
 
         self._must_show_form = False
         return self.async_show_form(
-            step_id="pairing_complete_import",
+            step_id=step_id,
             data_schema=vol.Schema({}),
             description_placeholders={"access_token": self._data[CONF_ACCESS_TOKEN]},
         )
 
-    async def async_step_tv_apps(
+    async def async_step_pairing_complete(
         self, user_input: Dict[str, Any] = None
     ) -> Dict[str, Any]:
-        """Handle app configuration to complete TV configuration."""
-        if user_input is not None:
-            if user_input.get(CONF_APPS_TO_INCLUDE_OR_EXCLUDE):
-                # Update stored apps with user entry config keys
-                self._apps[user_input[CONF_INCLUDE_OR_EXCLUDE].lower()] = user_input[
-                    CONF_APPS_TO_INCLUDE_OR_EXCLUDE
-                ].copy()
-
-            return await self._create_entry_if_unique(self._data)
+        """Complete non-import config flow by displaying final message to confirm pairing."""
+        return await self._pairing_complete("pairing_complete")
 
-        return self.async_show_form(
-            step_id="tv_apps",
-            data_schema=vol.Schema(
-                {
-                    vol.Optional(
-                        CONF_INCLUDE_OR_EXCLUDE, default=CONF_INCLUDE.title(),
-                    ): vol.In([CONF_INCLUDE.title(), CONF_EXCLUDE.title()]),
-                    vol.Optional(CONF_APPS_TO_INCLUDE_OR_EXCLUDE): cv.multi_select(
-                        VizioAsync.get_apps_list()
-                    ),
-                }
-            ),
-        )
+    async def async_step_pairing_complete_import(
+        self, user_input: Dict[str, Any] = None
+    ) -> Dict[str, Any]:
+        """Complete import config flow by displaying final message to show user access token and give further instructions."""
```

Comment:
Please break long strings at around 88 characters. The first sentence in a docstring should be a header line and should fit on one line. PEP8 says that docstrings should be limited at 72 characters line length.
https://www.python.org/dev/peps/pep-0008/#maximum-line-length